stress-tests/event-api/README.md

```
@@ -1 +1,84 @@
-test
+# Stress tests for event API
+
+Tests are written in TypeScript and are run by Node.js, provide outputs to console and files, analyze this outputs and provide a report. The tests can run in the CI pipeline, both one by one and in parallel. Written for the Kontur Events API. See more https://apps.kontur.io/events/swagger-ui/index.html?configUrl=/events/v3/api-docs/swagger-config#/. You can run just one test at once.
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)